tox.ini

```diff
@@ -8,6 +8,8 @@ setenv =
     GIT_COMMITTER_NAME = "test"
     GIT_AUTHOR_EMAIL = "test@example.com"
     GIT_COMMITTER_EMAIL = "test@example.com"
+    # git-lfs tests need HOME because git itself needs one
+    HOME={homedir)
```

Comment:
this isn't ~quite right as `HOME` means nothing on windows -- does adding `HOME` to `passenv` fix this?

Reply:
actually, now that I think about it, let's just set HOME for the tests in question and point it at a temporary directory -- that way the test suite doesn't modify global settings